apps/console/src/features/identity-providers/data/identity-provider-templates/templates/organization-enterprise-identity-provider/create-wizard-help.tsx

```diff
@@ -41,21 +42,31 @@ const OrganizationEnterpriseIdentityProviderCreateWizardHelp:
             ["data-testid"]: testId
         } = props;
 
+        const { t } = useTranslation();
+
         return (
             <>
                 <div data-testid={ testId }>
-                    <Heading as="h5">Name</Heading>
-                    <p>Provide a unique name for the enterprise authentication provider so that it can be
-                    easily identified.</p>
+                    <Heading as="h5">{
+                        t("console:develop.features.authenticationProvider.templates.organizationIDP.wizardHelp" +
+                            ".name.heading")
+                    }</Heading>
+                    <p>{ t("console:develop.features.authenticationProvider.templates.organizationIDP.wizardHelp" +
+                        ".name.description") }</p>
                     <p>E.g., MyOrgEnterpriseAuthProvider.</p>
                 </div>
 
                 <Divider/>
 
                 <div data-testid={ testId }>
-                    <Heading as="h5">Description</Heading>
-                    <p>Provide a description for the enterprise authentication provider to explain more about it.</p>
-                    <p>E.g., This is the authenticator for MyOrg, which acts as the IDP for MyApp.</p>
+                    <Heading as="h5">{
+                        t("console:develop.features.authenticationProvider.templates.organizationIDP.wizardHelp" +
+                            ".description.heading")
+                    }</Heading>
```

Comment:
```suggestion
                    <Heading as="h5">
                        { t("console:develop.features.authenticationProvider.templates.organizationIDP.wizardHelp" +
                            ".description. Heading") }
                    </Heading>
```

Reply:
Check the other places as well.

Reply:
Fixed in {
    "clientID": "h_fR9hMIt5wsEbio5EJ95w9Q15oa",
    "baseUrl": "https://api.asg.io/t/testasgardeo",
    "signInRedirectURL": "https://localhost:3000",
    "signOutRedirectURL": "https://localhost:3000",
    "scope": ["openid","profile"]
}